src/pubhub/pub_hub.bal

```diff
@@ -8,6 +9,18 @@ import ballerinax/java.jdbc;
 
 import maryamzi/websub.hub.mysqlstore;
 
+// Inbound auth handler for Hub service
+auth:InboundBasicAuthProvider inboundBasicAuthProvider = new;
+http:BasicAuthHandler inboundBasicAuthHandler = new(inboundBasicAuthProvider);
+
+// Outbound auth handler for Hub client
+auth:OutboundBasicAuthProvider OutBoundbasicAuthProvider = new({
```

Comment:
Do we need outbound for the hub? Since the hub only sends outbound requests to the subscribers?

Reply:
Removed